### ISSUE TITLE
x/tools/present: Docs say 'h' key in the browser will toggle extra emphasis of any highlighted lines, seemingly not true.

### ISSUE THREAD

**Issue by dmitshur** [golang/go#17375]
### What version of Go are you using (`go version`)?

```
go version go1.7.1 darwin/amd64
```
### What operating system and processor architecture are you using (`go env`)?

```
$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/Dmitri/Dropbox/Work/2013/GoLanding:/Users/Dmitri/Dropbox/Work/2013/GoLand"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
CC="clang"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/tw/kgz4v2kn4n7d7ryg5k_z3dk40000gn/T/go-build653232358=/tmp/go-build -gno-record-gcc-switches -fno-common"
CXX="clang++"
CGO_ENABLED="1"
```
### What did you do?

I read the documentation for present slide format at https://godoc.org/golang.org/x/tools/present.

It said:

> Also, inside the displayed text a line that ends
> 
> ```
> // HL
> ```
> 
> will be highlighted in the display; the 'h' key in the browser will toggle extra emphasis of any highlighted lines.
### What did you expect to see?

Pressing 'h' key in browser would have some effect on emphasis of highlighted lines.
### What did you see instead?

Pressing 'h' key has no effect on emphasis of highlighted lines.

Looking at the code,

https://github.com/golang/tools/blob/b5358b5feea9734b97ff1e792ca6bc55170bc92a/cmd/present/static/slides.js#L408

It says `'H' hides the help text`.

So something doesn't look right here. I don't know if the documentation is wrong, or if the implementation has a bug.

**Comment by ianlancetaylor** [comment]
CC @adg

**Comment by dmitshur** [comment]
> So something doesn't look right here. I don't know if the documentation is wrong, or if the implementation has a bug.

I did some code archeology to find out. The earlier commit I found where the 'h' key in browser was mentioned was the very first commit where the present format is added. It was actually in `talks` subrepo before being moved to `tools`. It was commit https://github.com/golang/talks/commit/a498aa1b8045be34f6e48e6c506368fd895a756a#diff-35e109b2aa892087b376d175ba51281dR93.

In that commit, I see no mention of 'h' key anywhere else except that one line of documentation.

Two years later, the 'h' key got mapped to hiding the help dialog in commit https://github.com/golang/tools/commit/753a094e8ab3f6e567ce75c7feebea04917a137f.

My best guess is this original feature was documented but never implemented. So removing it from documentation is the most appropriate fix. Will send a CL.
